bitbots_blackboard/bitbots_blackboard/capsules/world_model_capsule.py

```diff
@@ -392,6 +392,7 @@ def localization_pose_current(self) -> bool:
         """
         # if we can do this, we should be able to transform the ball
         # (unless the localization dies in the next 0.2 seconds)
+        return True
```

Comment:
I do not like merging this, as this might just lead to other errors down the line. Is there a better method?

Reply:
I fixed it by increasing the timeout ...

Reply:
I also checked the delay and it seems to be pretty low most of the time.